src/drive/web/modules/services/components/helpers.js

```diff
@@ -1,21 +1,53 @@
-import { models } from 'cozy-client'
-
 export const TYPE_DIRECTORY = 'directory'
+const FILE_TYPE = 'file'
+
+/**
+ * Is file param a correct note
+ *
+ * Inspired by https://github.com/cozy/cozy-client/blob/b7ffdd97d017bbd52226e0b61099da3090d8253f/packages/cozy-client/src/models/file.js#L52
+ * but don't check metadata (not provided while fetching Suggestions)
+ *
+ * @param {IOCozyFile} file io.cozy.files
+ * @returns {boolean}
+ */
+export const isNote = file => {
+  if (
+    file &&
+    file.name &&
+    file.name.endsWith('.cozy-note') &&
+    file.type === FILE_TYPE
```

Comment:
you can edit this helper since you have more informations now (metadata.title & metadata.version)

Reply:
I'm wondering if we should not change the helper from cozy-client directly instead of having this one in Drive.

Reply:
6 usages in Cozy:

=> 5 in Drive, 1 in Cozy UI :

https://cs.github.com/?scope=org%3Acozy&scopeName=cozy&q=models.file.isNote%28

Reply:
I know and this is exactly my point... We learnt that requesting metadata.content it's big. So we should not request it anymore. And we should not relying on it to know if the files is a note or not a note. No?